src/enzo/EvolvePhotons.C

```diff
@@ -716,6 +719,28 @@ int EvolvePhotons(TopGridData *MetaData, LevelHierarchyEntry *LevelArray[],
   }
 #endif
 
+  /* Double check and clean up photon sources that have expired */
+
+  if (NumberOfSources > 0){
+    RS = GlobalRadiationSources->NextSource;
+
+    /* Clean up photon sources if time is up */
+    while (RS != NULL){
+      if ( ((RS->CreationTime + RS->LifeTime) < PhotonTime) && LoopTime == TRUE &&
+	           DeleteSources) {
+      if (debug) {
```

Comment:
This is a real nitpick, but the tabbing here and with the print statements is a bit inconsistent. Could you line those up?

Reply:
Done!